nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -8,31 +8,33 @@ tags:
 - Configuration
 ---
 
-The behavior of each persister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing available properties, or using code API. The exact same settings are exposed via `app.config` file and code configuration API.
+In NServiceBus.Azure the behavior of the AzureStoragePersister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing the available properties, or by using the code API. The exact same settings are exposed via the `app.config` and the code configuration API.
```

Comment:
Should `NServiceBus.Azure` and `AzureStoragePersister` be formatted as code?

"respective configuration sections" is confusing here as we don't know which sections and with respect to what. I realize that this context is given below. Maybe at this point, all that is needed is "can be controlled by setting properties in the `app.config` file, or by using the code API."

These two sentences seem to repeat themselves. May drop ", or by using the code API." and making the second sentence more focused on the fact that code API also provides access to those same properties.